config/settings.yml

```diff
@@ -192,6 +192,12 @@ defaults: &defaults
       key: <%= certificate('broker.key') %>
       cert: <%= certificate('broker.crt') %>
   ####################
+  # ETCD SETTINGS #
```

Comment:
We would ideally have different settings for each process as discussed. Most likely etcd would be common for all processes, still would make sense to see that segregation. Not sure if this is in scope of this pr. If not, it’s fine.

Reply:
Will make those changes for the backup manager pr